concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -269,6 +269,12 @@ resources:
       versioned_file: bosh-runtime-config-vars-store.yml
       initial_version: "-"
 
+  - name: concourse-bosh-deployment
+    type: git
+    source:
+      uri: https://github.com/concourse/concourse-bosh-deployment
+      tag_filter: "v4.2.*"
```

Comment:
Can we pin the exact version of concourse we're using. We currently rely on the property that deploying a specific revision of paas-bootstrap will result in a consistently configured setup whenever it's run.